<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrist</class>
 <widget class="QWidget" name="MainFrist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_gun_advert">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>800</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>词条滚动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>291</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_city">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>city</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_date">
      <property name="text">
       <string>日期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_time">
      <property name="text">
       <string>时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_temperature">
      <property name="text">
       <string>temperature</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_weather_icon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>weather_icon</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_weather">
        <property name="text">
         <string>weather_</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_hemidity">
      <property name="text">
       <string>humidity</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="lighting_btn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>360</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>openled</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openCar_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>360</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>打开监控</string>
   </property>
  </widget>
  <widget class="QPushButton" name="property_btn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>420</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>物业服务</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>480</width>
     <height>180</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mp4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>480</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>视频</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>420</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>关机</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
